pkg/controller/druid/handler.go

```diff
@@ -879,6 +922,11 @@ func namespacedName(name, namespace string) *types.NamespacedName {
 	return &types.NamespacedName{Name: name, Namespace: namespace}
 }
 
+// IntPointer returns a pointer to int32.
+func IntPointer(i int32) *int32 {
+	return &i
+}
+
```

Comment:
```suggestion
```

Reply:
sorry i am having a bad day at work.

Reply:
How come now CI is failing ? Ran go fmt local it works.

Reply:
gofmt did not like the extra empy line introduced at line#925 after  latest commit.